try {
     do something that might cause an exception
catch(ExpectedException e) {
     throw new DifferetnException(e);
}